content/src/main/content/jcr_root/apps/acs-samples/angularjs-integration/clientlibs/footerLib/js/angular/core/main.js

```diff
@@ -0,0 +1,30 @@
+/*global angular*/
+
+/*
+ * This is the definition of the application module itself as well as core configurations like routing and the http service.
+ * Since this file defines that module that all of our controllers and services are part of, it needs to be listed first in the clientlibs js.txt file.
+ */
+var sampleModule = angular.module('angularjs-sample', ['ngRoute']);
+
+/*
+ * The routeProvider configuration maps URL paths to AJAX content to be loaded into the ng-view container defined in the containerPage component.
+ * Note that the templateUrls are each using a .content Sling selector.  This allows us to switch out the content portion of the containerPage with that of the partials.
+ */
```

Comment:
This is 100% sling.  Angular has no concept of selectors.  I updated the
comment to make this more clear.  Let me know if it is still confusing.

On Sun, Mar 8, 2015 at 8:45 AM, david g. <notifications@github.com> wrote:

> In
> content/src/main/content/jcr_root/apps/acs-samples/angularjs-integration/clientlibs/footerLib/js/angular/core/main.js
> <https://github.com/Adobe-Consulting-Services/acs-aem-samples/pull/26#discussion_r26006199>
> :
>
> > @@ -0,0 +1,30 @@
> > +/*global angular*/
> > +
> > +/*
> > + * This is the definition of the application module itself as well as core configurations like routing and the http service.
> > + * Since this file defines that module that all of our controllers and services are part of, it needs to be listed first in the clientlibs js.txt file.
> > + */
> > +var sampleModule = angular.module('angularjs-sample', ['ngRoute']);
> > +
> > +/*
> > + * The routeProvider configuration maps URL paths to AJAX content to be loaded into the ng-view container defined in the containerPage component.
> > + * Note that the templateUrls are each using a .content Sling selector.  This allows us to switch out the content portion of the containerPage with that of the partials.
> > + */
>
> I think its worth noting that the .content selector is a custom
> abstraction defined by the implementation of the angularjs-integration
> base-page, and not some "magical" AEM selector.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Adobe-Consulting-Services/acs-aem-samples/pull/26/files#r26006199>
> .
>